libs/products/src/lib/products.ts

```diff
@@ -15,4 +15,6 @@ export const exampleProducts: Product[] = [
     name: 'Product 2',
     price: 200,
   },
-];
\ No newline at end of file
+];
+
+export const products = () => "products";
```

Comment:
At `poducts.spec.tsx` spec to have a function that didn't exist in products file so i created it.